anonymize.sql

```diff
@@ -0,0 +1,54 @@
+CREATE EXTENSION pgcrypto;
+
+UPDATE booking SET "token" = substring(md5(random()::text),1 , 6) WHERE "token" is not null;
+UPDATE deposit SET "amount" = random() WHERE "amount" is not null;
+UPDATE mediation SET "credit" = 'credit text' WHERE "credit" is not null;
+UPDATE offer SET "bookingEmail" = 'ano@nym.ized' WHERE "bookingEmail" is not null;
+UPDATE offerer SET "validationToken" = substring(md5(random()::text),1 , 27) WHERE "validationToken" is not null;
+UPDATE offerer SET "name" = 'offerer_' || "id";
+UPDATE offerer SET "siren" = trunc(random() * (999999999-100000000+1) + 100000000)::int WHERE "siren" is not null;
+UPDATE offerer SET "address" = md5(random()::text),
+  "postalCode" = floor(random() * (99999-10000+1) + 10000)::int,
+  "city" = md5(random()::text) WHERE "address" is not null;
+UPDATE offerer SET "iban" = trunc(random() * (999999999-100000000+1) + 100000000)::int WHERE "iban" is not null;
+UPDATE offerer SET "bic" = trunc(random() * (999999999-100000000+1) + 100000000)::int WHERE "bic" is not null;
+
+UPDATE payment SET "recipientName" = 'recipient_name';
+UPDATE payment SET "iban" = trunc(random() * (999999999-100000000+1) + 100000000)::int;
+UPDATE payment SET "bic" = trunc(random() * (999999999-100000000+1) + 100000000)::int;
+UPDATE payment SET "amount" = 1 WHERE "amount" > 0;
+UPDATE payment SET "recipientSiren" = trunc(random() * (999999999-100000000+1) + 100000000)::int WHERE "recipientSiren" is not null;
+UPDATE payment SET "reimbursementRate" = trunc(random() * (9 - 1 + 1) + 1)::int WHERE "recipientSiren" is not null;
```

Comment:
Les taux de remboursement sont déterministes selon le type de l'offre (et selon des règles publiques). Je ne pense pas que ce soit une bonne idée de les randomiser

Reply:
C'est noté !